src/eviction/lru.c

```diff
@@ -636,14 +731,19 @@ uint32_t evp_lru_req_clines(struct ocf_request *req,
 
 	evp = req->io_queue->eviction_idx++ % OCF_NUM_EVICTION_LISTS;
 
-	lru_iter_eviction_init(&iter, cache, part, evp, cl_write_lock, req);
+	lru_iter_eviction_init(&iter, cache, part, evp, req);
 
 	i = 0;
 	while (i < cline_no) {
 		if (!evp_lru_can_evict(cache))
 			break;
 
-		cline = lru_iter_eviction_next(&iter, &core_id, &core_line);
+		if (part_id != PARTITION_INVALID) {
```

Comment:
Here too. TBH I didn't find a single use if `PARTITION_INVALID` that would really mean "invalid".